domain/src/main/kotlin/no/nav/su/se/bakover/domain/vilkår/Vilkår.kt

```diff
@@ -550,6 +567,7 @@ sealed class Vilkårsvurderinger {
                     is InstitusjonsoppholdVilkår,
                     is LovligOppholdVilkår,
                     is PersonligOppmøteVilkår,
+                    is FamiliegjenforeningVilkår,
```

Comment:
Ikke veldig viktig, men prøvde å skille de vilkårene som er tenkt at vi skal kunne bruke for uføre de som aldri skal brukes for uføre når jeg la til pensjon, kunne vurdert om du ville følge det.